README.md

```diff
@@ -41,4 +41,5 @@ Findbugs Plugin version|Embedded SpotBugs/Findbugs version|Embedded Findsecbugs
 3.11.0                 | 3.1.12 (SpotBugs)                | 1.8.0                      | 7.4.3sb                   | 1.8|7.6~|5.10.1.16922
 4.0.0                  | 4.0.0 (SpotBugs)                 | 1.10.1                     | 7.4.7 (sb-contrib)        | 1.8|7.6~|5.10.1.16922
 4.0.1                  | 4.1.2 (SpotBugs)                 | 1.10.1                     | 7.4.7 (sb-contrib)        | 1.8|7.6~|5.10.1.16922
-4.0.2-SNAPSHOT         | 4.1.2 (SpotBugs)                 | 1.10.1                     | 7.4.7 (sb-contrib)        | 1.8|7.6~|5.10.1.16922
\ No newline at end of file
+4.0.2                  | 4.1.4 (SpotBugs)                 | 1.10.1                     | 7.4.7 (sb-contrib)        | 1.8|7.6~|5.10.1.16922
```

Comment:
Sorry I forgot requesting you to handle, but the version of `findsecbugs` for `sonar-findbugs 4.0.2` is needed to be updated too.

Reply:
No problem. I'm not a maven expert, but I think it's done.